### PR TITLE
pre-reqs: Update nydus snapshotter version

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Bump the nydus snapshotter version to v0.13.13, in order to pick the nydus-snapshotter caching workaround
https://github.com/containerd/nydus-snapshotter/pull/593

### CODE REVIEW

**Review by fidencio — APPROVED** [review]
lgtm, thanks @stevenhorsman!

File: install/pre-install-payload/payload.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ official_containerd_version=${official_containerd_version:-"1.7.7"}
 vfio_gpu_containerd_repo=${vfio_gpu_containerd_repo:-"https://github.com/confidential-containers/containerd"}
 vfio_gpu_containerd_version=${vfio_gpu_containerd_version:-"1.7.0.0"}
 nydus_snapshotter_repo=${nydus_snapshotter_repo:-"https://github.com/containerd/nydus-snapshotter"}
-nydus_snapshotter_version=${nydus_snapshotter_version:-"v0.13.3-multiarch"}
+nydus_snapshotter_version=${nydus_snapshotter_version:-"v0.13.13"}
```
Comment (ldoktor) [inline install/pre-install-payload/payload.sh:16]: is it necessary to keep updating the default when we have it specified in the `Makefile`? Having 2 places can be confusing...

Comment (stevenhorsman) [reply install/pre-install-payload/payload.sh:16]: I agree with you, but didn't want to go into that on this PR. Do you thing an approach of us letting the `set -o nounset` fail if it's not passed in would make sense?

Comment (stevenhorsman) [reply install/pre-install-payload/payload.sh]: @ldoktor - I've added a second commit, so please check that out and see if you are happy and we can get @fidencio to re-review too.

**Review by mkulke — APPROVED** [review]
tested it, the nydus version installed by the operator is able to perform guest-pulls, even if images have been pulled by runc on the node before (pause). vice versa also works.

**Review by ldoktor — APPROVED** [review]
Thanks, lgtm now.

**Review by fidencio — APPROVED** [review]
yay!

**Pull request merged**